### ISSUE TITLE
CoolProp 6.4.2dev0, MATLAB wrapper with Python 3.9

### ISSUE THREAD

**Issue by smyng91** [CoolProp/CoolProp#2126]
# Description

CoolProp 6.4.2dev0 (Python 3.9) compiled from source loads properly in MATLAB R2022a but when you run

```Matlab
 py.CoolProp.CoolProp.PropsSI('T','P',101325,'Q',0,'Water')
```
it yields the following error:
```Matlab
Unable to resolve the name 'py.CoolProp.CoolProp.PropsSI'.
```
*This approach works properly with CoolProp 6.4.1

# Versions

**CoolProp Version:** 6.4.2dev0
**Operating System and Version:** macosx-11.0-arm64
**Access Method:** MATLAB with Python 3.9

**Comment by ibell** [comment]
How did you resolve this issue?

**Comment by smyng91** [comment]
Solved by installing [Python 3.8.10](https://www.python.org/downloads/release/python-3810/) and running
```matlab
pyenv('Version','/usr/local/bin/python3.8')
[v,e] = pyversion; system([e,' -m pip install --user -U CoolProp'])
```
It seems like MATLAB is having issues linking to conda and pyenv versions of Python.

**Comment by ibell** [comment]
Please see https://github.com/CoolProp/CoolProp/issues/2119
